{{cookiecutter.project_shortname}}/scripts/server

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
+script_path=$(dirname "$0")
+
+export FLASK_ENV=development
+export INVENIO_SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://{{cookiecutter.project_shortname}}:{{cookiecutter.project_shortname}}@localhost/{{cookiecutter.project_shortname}}"
+export INVENIO_THEME_LOGO='images/logo.svg'
+
+# Start Worker and Server
+pipenv run celery worker -A invenio_app.celery -l INFO & pid_celery=$!
+
+pipenv run invenio run \
+       --cert "$script_path"/../docker/nginx/test.crt \
+       --key "$script_path"/../docker/nginx/test.key & pid_server=$!
+
+trap 'kill $pid_celery $pid_server &>/dev/null' EXIT
+
+wait $pid_celery $pid_server
```

Comment:
Linked approach https://github.com/galterlibrary/InvenioRDM-at-NU/blob/master/scripts/server, allows pdb sessions to work correctly and may solve invenio-cli#19

Reply:
I like it, but since we should deprecate/remove the scripts I would avoid making work on this. What do you think?

Reply:
If we are deprecating the script then keep this for now and perhaps look into re-using the other approach in the CLI in the future.